phpcs-ruleset.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0"?>
+<ruleset name="Spryker">
```

Comment:
Why duplicating the file?
I am not a big fan of unDRY code etc

Better to find a way to reuse existing files and stuff.